pypi_formula_mappings.json

```
@@ -182,6 +182,9 @@
   "jenkins-job-builder": {
     "exclude_packages": ["six"]
   },
+  "jinja2-cli": {
+    "extra_packages": ["pyyaml", "toml", "xmltodict"]
+  },
```

I think feature dependencies defined by `extras_require` can be enabled via common pip install syntax, which better tracks any changes upstream may do:
```suggestion
  "jinja2-cli": "jinja2-cli[yaml,toml,xml]",
```

---

Ah, I did not know that! Will adjust my PR.